qiskit/providers/aer/noise/noise_model.py

```diff
@@ -810,22 +808,20 @@ def to_dict(self, serializable=False):
 
         # Add specific qubit errors
         for name, qubit_dict in self._local_quantum_errors.items():
-            for qubits_str, error in qubit_dict.items():
+            for qubits, error in qubit_dict.items():
                 error_dict = error.to_dict()
                 error_dict["operations"] = [name]
-                error_dict["gate_qubits"] = [self._str2qubits(qubits_str)]
+                error_dict["gate_qubits"] = [list(qubits)]
                 error_list.append(error_dict)
 
         # Add non-local errors
         for name, qubit_dict in self._nonlocal_quantum_errors.items():
-            for qubits_str, noise_qubit_dict in qubit_dict.items():
-                for noise_qubits_str, error in noise_qubit_dict.items():
+            for qubits, noise_qubit_dict in qubit_dict.items():
+                for noise_qubits, error in noise_qubit_dict.items():
                     error_dict = error.to_dict()
                     error_dict["operations"] = [name]
-                    error_dict["gate_qubits"] = [self._str2qubits(qubits_str)]
-                    error_dict["noise_qubits"] = [
-                        self._str2qubits(noise_qubits_str)
-                    ]
+                    error_dict["gate_qubits"] = [list(qubits)]
```

Comment:
```suggestion
                    error_dict["gate_qubits"] = [qubits]
```
As jake said, might as well leave them as tuples